.github/workflows/lint.yml

```diff
@@ -0,0 +1,27 @@
+name: lint
```

Comment:
This connector apparently has `build` and `lint` actions in one. I don't mind, but then we don't need this one, we should remove one of the lint actions.